src/main/java/pl/javastart/task/app/Main.java

```diff
@@ -0,0 +1,14 @@
+package pl.javastart.task.app;
+
+public class Main {
+
+    public static void main(String[] args) {
+        CashController cashController = new CashController();
+        cashController.getTicketCash().addEvent("Koncert Metallica", "Warszawa", 100);
+        cashController.getTicketCash().addEvent("Regaty morskie", "Gdańsk", 200);
+        cashController.getTicketCash().addEvent("Pokaz sztucznych ogni", "Gdynia", 160);
+        cashController.getTicketCash().addEvent("Pyrkon", "Poznań", 300);
+        cashController.getTicketCash().addEvent("Poznań Game Arena", "Poznań", 200);
```

Comment:
To wygląda dziwnie. Mamy tutaj klasę serwisową `CashController`, a z niej pobieramy kolejną klasę serwisową `TicketCash`. W takich przypadkach raczej dostarcza się funkcjonalność przez klasę wyższego rzędu, a w implementacji ukrywa się z czego ona korzysta. Czyli oczekiwany wygląd wywołania tutaj to:

```
cashController.addEvent("Koncert Metallica", "Warszawa", 100);
```

Natomiast to, że cachController przekazuje to dalej do `TicketCash` niech pozostanie z tego punktu widzenia ukryte.